Platform/ArrowlakeBoardPkg/Library/FspsUpdUpdateLib/FspsUpdUpdateLib.c

```diff
@@ -631,6 +621,18 @@ UpdateFspConfig (
   FspsConfig->AmtEnabled = 0x1;
   FspsConfig->EnableTcoTimer = 0x1;
 
+  if (MtlIsSocM ()) {
```

Comment:
Is ARL and MTL sharing the same function? Would be good if this function is named ArlIsSocM instead but if can't then it's fine

Reply:
currently the same tagging MtlIsSocM is used for both MTL & ARL in bios source code, so we didnt change it yet to avoid any mismatch